src/logging/multiplexing_logger.cpp

```diff
@@ -21,9 +21,15 @@
 
 namespace mpl = multipass::logging;
 
+multipass::logging::MultiplexingLogger::MultiplexingLogger(UPtr system_logger)
+    : system_logger{std::move(system_logger)}
+{
+}
+
 void mpl::MultiplexingLogger::log(mpl::Level level, CString category, CString message) const
 {
     std::shared_lock<decltype(mutex)> lock{mutex};
+    system_logger->log(level, category, message);
```

Comment:
Just thinking if we should add the raw pointer to `system_logger` to the `loggers` array in the MultiplexingLogger constructor and then not worry about the extra `log()` call here.

Not going to block on that though.

Reply:
I could, but saving copying the raw pointer out of the unique_ptr to another location defeats the safety the unique_ptr gives. So I'd rather not.

Reply:
Ok, works for me:)